src/KafkaFlow.Retry.Postgres/RetryQueueDataProvider.cs

```diff
@@ -107,6 +107,11 @@ public async Task<QueuePendingItemsResult> CheckQueuePendingItemsAsync(QueuePend
         }
     }
 
+    public Task<long> CountQueuesAsync(CountQueuesInput input)
+    {
+        throw new NotImplementedException();
```

Comment:
I think it would be best to implement this before shipping this feature. @alek-github can you please help here since you are more comfortable with Postgres?

Reply:
We can deliver the MongoDB and SQL Server by now. The Postgres could be done afterwards.

Reply:
Closing this as it was implemented.